firefox_ui_tests/remote/manifest.ini

```diff
@@ -1 +1,4 @@
+[DEFAULT]
+remote = true
```

Comment:
That's a left-over when I tried to specify the tests to run via the `--type` option. Given that wasn't working and I removed this, we can also remove those entries for now. But what I would like to see are types like `security` which we could stick to functional and remote security tests, so that you can fine-control which tests to run.